### DISCUSSION TITLE
Does this tool supports Oracle ? Can I use the utility from another python class?

### DISCUSSION THREAD

**Discussion by sanket1203** [agronholm/sqlacodegen#180] [Ideas]
Hi Does this utility supports fro oracle database?
Can I use the utility from another python class?

**Comment by agronholm** [comment]
It doesn't have any special code paths for Oracle, but it should support any RDBMS supported by SQLAlchemy.

Yes, it can be used programmatically. See [main.py](https://github.com/agronholm/sqlacodegen/blob/2.3.0/sqlacodegen/main.py) for an example.
